src/main/java/nl/novi/techiteasy1121/services/TelevisionService.java

```diff
@@ -167,6 +162,44 @@ public TelevisionDto transferToDto(Television television){
         dto.setOriginalStock(television.getOriginalStock());
         dto.setSold(television.getSold());
 
+        // Als extra op deze transfer methode, voegen we ook de relaties toe.
+        // Hier moeten we eerst een null check voor doen,
+        // omdat we anders in CIModule.transferToDto de get-methodes van "null" aanroepen en dat kan niet.
```

Comment:
Bij het omgaan met de relaties in je Dto's is het goed dat je null-checks uitvoert. Dit verhoogt de stabiliteit van je applicatie. Blijf dit zo doen!